application/templates/cms/measure_actions.html

```diff
@@ -27,11 +27,11 @@
                     <div class="delete_version" id='delete-version-{{ measure.guid }}'>
                         <div class="multiple-choice" style="display:block">
                             <input id="delete-radio-yes-{{ measure.guid }}" type="radio" name="radio-group" value="Yes">
-                            <label for="delete-radio-yes-{{ measure.guid }}">Delete</label>
+                            <label for="delete-radio-yes-{{ measure.guid }}">delete</label>
                         </div>
                         <div class="multiple-choice">
                             <input id="delete-radio-no-{{ measure.guid }}" type="radio" name="radio-group" value="No">
-                            <label for="delete-radio-no-{{ measure.guid }}">Cancel</label>
+                            <label for="delete-radio-no-{{ measure.guid }}">go back</label>
```

Comment:
'Don't delete' doesn't fit on one line @TheDoubleK as that's what I wanted initially. I'll chat to the guys but it might be clarity with words wins over clean design.

Reply:
👍 done.